DataDog/integrations-core
# [elastic] elasticsearch reports i/o stats differently as of version 5.0

### What does this PR do?

This change accounts for the fact that some disk I/O metrics are reported differently by the Elasticsearch nodes stats API as of Elasticsearch version 5.0.0. See #972.

### Motivation

I recently upgraded Elasticsearch and miss having disk I/O metrics in Datadog.

### Testing Guidelines

I tested this change in a production deployment, and it worked as expected. I attempted to update the tests as part of this PR, but I was unable to get the tests to actually run. I added a line to `.travis.yml` in the hopes that Travis will run the tests using an Elasticsearch version 5.0.0 installation.

### Versioning

- [ ] Bumped the version check in `manifest.json` [N/A because no new metrics are being introduced]
  for the new section.

### Additional Notes

This PR doesn't create any additional metrics. It merely fixes the following metrics for Elasticsearch installations whose version is >= 5.0.0:

* `elasticsearch.fs.total.disk_reads`
* `elasticsearch.fs.total.disk_writes`
* `elasticsearch.fs.total.disk_io_op`
* `elasticsearch.fs.total.disk_read_size_in_bytes`
* `elasticsearch.fs.total.disk_write_size_in_bytes`

This PR does not fix the `elasticsearch.fs.total.disk_io_size_in_bytes` metric because that metric is no longer readily available from the Elasticsearch nodes stats API.
